packages/aws-cdk/bin/cdk.ts

```diff
@@ -124,6 +124,13 @@ async function parseCommandLineArguments() {
         desc: 'Continuously observe the project files, ' +
           'and deploy the given stack(s) automatically when changes are detected. ' +
           'Implies --hotswap by default',
+      })
+      .options('logs', {
+        type: 'boolean',
+        default: true,
+        desc: 'Show CloudWatch log events from hotswapped resources in the terminal. ' +
+          "'true' by default, use --no-logs to turn off. " +
+          "Only in effect if '--watch' option is used",
```

Comment:
The "hotswapped" part is not true anymore, we probably should use the same wording we do in the ReadMe:

```suggestion
        desc: 'Show CloudWatch log events from all resources in the selected Stacks in the terminal. ' +
          "'true' by default, use --no-logs to turn off. " +
          "Only in effect if specified alongside the '--watch' option",
```